core/src/main/java/de/danoeh/antennapod/core/preferences/UserPreferences.java

```diff
@@ -1057,4 +1059,15 @@ public static void setFeedFilter(String value) {
                 .apply();
     }
 
+    public static long getUsageCountingMillis() {
+        return prefs.getLong(PREF_USAGE_COUNTING_DATE, -1);
+    }
+
+    private static void setUsageCountingMillis(long value) {
```

Comment:
I would combine set/reset to a single method. Can't think of a case where we want to use set without reset.